.github/workflows/julia-auto.yml

```diff
@@ -0,0 +1,80 @@
+name: hdf5 dev autotools julia
+
+on:
+  workflow_call:
+    inputs:
+      build_mode:
+        description: "release vs. debug build"
+        required: true
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  julia_build_and_test:
+    name: "julia ${{ inputs.build_mode }}"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Sources
+        uses: actions/checkout@v4.1.1
+
+      - name: Install Dependencies
+        shell: bash
+        run: |
+          sudo apt-get update
+          sudo apt-get install autoconf automake libtool libtool-bin libaec-dev
+          sudo apt-get install doxygen graphviz
+          sudo apt install -y zlib1g-dev libcurl4-openssl-dev libjpeg-dev wget curl bzip2
+          sudo apt install -y m4 flex bison cmake libzip-dev openssl build-essential
+
+      - name: Autotools Configure
+        shell: bash
+        run: |
+          sh ./autogen.sh
+          mkdir "${{ runner.workspace }}/build"
+          cd "${{ runner.workspace }}/build"
+          $GITHUB_WORKSPACE/configure \
+            FCFLAGS="-fPIC -fortranlibs" \
```

Comment:
Doesn't seem like we need to set FCFLAGS here since we disable fortran below

Reply:
Agree. Should remove this line.